File: EncodeAndDecodeStrings.java
import java.util.List;
class Solution {
    char[] code = new char[256];
    for(int i = 0; i < code.length; i++){
        char character = (char) (256 - code[i]);
        code[i] = character;
    }

    public String encode(List<String> strs) {
        String encodedString = "";
        for (String s : strs){


        }
        return encodedString;
    }

    public List<String> decode(String str) {

    }
}

File: encode_and_decode_strings.py
import code


class Solution:
    def encode(self, strs: list[str]) -> str:
        encoded_string = ""
        for s in strs:
            encoded_string += str(len(s)) + "#" + s
        return encoded_string

    def decode(self, encoded_string: str) -> list[str]:
        length_as_string = ""
        length_as_int = 0
        res = []
        i = 0
        while i != "#":
            length_as_string += encoded_string[i]
            i += 1
        length_as_int = int(length_as_int)
        word = ""
        for i in range(i, length_as_int + 1):
            word += encoded_string[i]
        res.append(word)
        return res


sol = Solution()
codestring = sol.encode(["Hello", "World"])
print(sol.decode(codestring))
